app/Http/Controllers/QuizAPIController.php

```diff
@@ -18,10 +18,39 @@ function getQuizzes()
                 'message' => 'Quizzes retrieved',
                 'data' => $quizzes
             ], 200);
-        } catch (\Exception) {
+        } catch (Exception) {
```

Comment:
```suggestion
        } catch (Exception $e) {
```
Make a variable for the exception even if you don't use it